docs/resources/cloud_init_disk.md

```diff
@@ -47,12 +52,17 @@ EOT
 resource "proxmox_vm_qemu" "vm" {
 ....
   // Define a disk block with media type cdrom which reference the generated cloud-init disk
-  disk {
-    type    = "scsi"
-    media   = "cdrom"
-    storage = local.iso_storage_pool
-    volume  = proxmox_cloud_init_disk.ci.id
-    size    = proxmox_cloud_init_disk.ci.size
+  disks {
+    scsi {
+      scsi0 {
+        disk {
```

Comment:
With the new disk schema it should be like this:

```hcl
cd {
  iso = "${local.iso_storage_pool}:${proxmox_cloud_init_disk.ci.id}
}
```

Not 100% sure what `local.iso_storage_pool` and `proxmox_cloud_init_disk.ci.id` are in this situation.